Program.cs

```diff
@@ -15,50 +24,61 @@ static void Main(string[] args)
             do{
                 Console.WriteLine("Select the type of staff\n1)Teacher\n2)Administrator\n3)Support\n4)QUIT");
                 staffOptions=Convert.ToInt32(Console.ReadLine());
-                switch(staffOptions){
-                    case 1: 
-                        Staff teacher=new Teacher();
-                        Program.MainActions(teacher);
-                        break;
-                    case 2: 
-                        Staff administrator=new Administrator() ;
-                        Program.MainActions(administrator);
-                        break;
-                    case 3: 
-                        Staff support=new Support() ;
-                        Program.MainActions(support);
-                        break;
-                    case 4: 
-                        continueStaffSelection=false;
-                        break;
-                }
+               
+                    switch(staffOptions){
+                        case 1: 
+                            Program.MainActions("Teacher");
+                            break;
+                        case 2: 
+                            Program.MainActions("Administrator");
+                            break;
+                        case 3: 
+                            Program.MainActions("Support");
+                            break;
+                        case 4:
+                            continueStaffSelection=false;
+                            break;
+                        }
             }while(continueStaffSelection);
         }
 
-        private static void MainActions(Staff staff){
+
+        private static void MainActions(String staffType){
+
             int mainOptions;
-            bool continueMainActions=true;
-            string staffType=staff.GetType().Name.ToUpper();
+            bool continueMainActions = true;
+           
             do{
-                Console.WriteLine("WLECOME TO {0} STAFF PORTAL\nselect Options\n1)Add staff\n2)Delete staff\n3)Update staff details\n4)View staff details",staffType);
-                mainOptions=Convert.ToInt32(Console.ReadLine());
-                switch(mainOptions){
-                    case 1: 
-                        staff.AddStaff();
+                Console.WriteLine("WLECOME TO {0} STAFF PORTAL\nselect Options\n1)Add staff\n2)Delete staff\n3)Update staff details\n4)View staff details", staffType);
+                mainOptions = Convert.ToInt32(Console.ReadLine());
+                switch (mainOptions){
+                    case 1:
+                        Program.StaffId++;
+                        Staff staff = HelperMethods.StaffAdd(staffType, Program.StaffId,Program.staffs);
```

Comment:
Instead of naming as 'helpermethods' , it would be more developer-friendly if you mention for which model this helper class is used, say like 'StaffHelper'